src/tooltip/test/tooltip.spec.js

```diff
@@ -853,9 +938,9 @@ describe( '$tooltipProvider', function() {
         elm.trigger('fakeTrigger');
         expect( tooltipScope.isOpen ).toBeTruthy();
         elm.trigger('fakeTrigger');
+        $timeout.flush();
```

Comment:
Yes, the test was failing on my branch since there is a timeout for closing the popup.